index.user.js

```diff
@@ -75,14 +75,53 @@
          }
         `
       ]
+    },
+    enforceLatestTweets: {
+      default: true,
+      test: ({ parsedUrl }) => {
+        const { pathname } = parsedUrl
+        return pathname === '/' || pathname === '/home'
+      },
+      init: () => {
+        waitUntil(() => document.querySelector('[data-testid="primaryColumn"]'), 500)
+          .then(timeline => {
+            let lastTime = null
+            const isShowingLatest =
+              document.documentElement.getAttribute('lang') === 'en'
+                ? document.title.startsWith('Latest')
+                : [].every.call(
+                    document.querySelectorAll('time'),
+                    time => {
+                      const currentTime = new Date(time.getAttribute('datetime'))
+                      const isChronological = !lastTime || lastTime > currentTime
+                      lastTime = currentTime
+                      return isChronological
+                    }
+                  )
+
+            if (!isShowingLatest) {
+              timeline.querySelector('[role="heading"]')
+                .parentNode.parentNode.querySelector('[role="button"]')
+                .click()
+
+              setTimeout(() =>
+                document.querySelector('[role="menu"] [role="menuitem"]')
+                .click()
```

Comment:
I was thinking that instead of clicking the button (could be fragile) maybe there is a way to set that with a post request to some endpoint. I guess that the current solution is ok-ish too